protoc

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+if [ `uname -s` = "Darwin" ]; then
+  os=osx
+else
+  os=linux
+fi
+arch=`uname -m`
+
+protocbin=.protoc
+protocversion=3.0.0
+protocurl=https://github.com/google/protobuf/releases/download/v$protocversion/protoc-$protocversion-$os-$arch.zip
+
+if [ ! -f $protocbin ]; then
+  echo "downloading $protocbin" >&2
+  tmp=`mktemp -d -t protoc.XXX`
+  curl -L --silent --fail -o $tmp/$protocbin.zip $protocurl
+  unzip -q $tmp/$protocbin.zip -d $tmp
```

Comment:
it would be Nice to check the sha of the binary (like we do for sbtlaunch) to check i.e. if the urlc hangges.